sdk/spring/spring-reference.yml

```diff
@@ -1,1519 +0,0 @@
-# Please refer to this link to see design details and how to change:
-# https://dev.azure.com/SpringOnAzure/Spring%20on%20Azure/_wiki/wikis/spring-integration-private.wiki/280/Spring-Reference-for-Intellij-Plugin
----
```

Comment:
Hi, @saragluna , is this fil still been used? If it's still been used, I'll revert this file.